packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -568,7 +568,14 @@ export class RunningSummarizer extends TypedEventEmitter<ISummarizerEvents> impl
 				};
 				const summarizeResult = this.generator.summarize(summaryOptions, resultsBuilder);
 				// ensure we wait till the end of the process
-				return summarizeResult.receivedSummaryAckOrNack;
+				const result = await summarizeResult.receivedSummaryAckOrNack;
+				if (!result.success) {
+					this.mc.logger.sendErrorEvent({
+						eventName: "SummarizeFailed",
+						maxAttempts: 1,
+						summaryAttempts: 1,
+					});
+				}
```

Comment:
mostly unrelated to this change, but should we call `this.stopSummarizerCallback("failToSummarize");` in this case as well?

Reply:
This case is hit when we're already closing the summarizer or we tried an on-demand summary